README.md

```diff
@@ -115,6 +115,8 @@ Uncomment the `Buy Tokens` sections in `packages/nextjs/app/token-vendor/page.ts
 
 > ✏️ We can't hard code the vendor address like we did above when deploying to the network because we won't know the vendor address at the time we create the token contract.
 
+> ✏️ So instead, edit `YourToken.cairo` to mint the tokens to the `recipient` (deployer) in the **constructor()**.
```

Comment:
We already said this instructions lines above
`Mint **2000** (\* 10 \*\* 18) to your frontend address using the constructor()`

So the same instructions repeats here. What do you suggest?